exec/java-exec/src/main/java/org/apache/drill/exec/planner/logical/DrillAnalyzeRel.java

```diff
@@ -37,8 +37,11 @@
  */
 public class DrillAnalyzeRel extends SingleRel implements DrillRel {
 
-  public DrillAnalyzeRel(RelOptCluster cluster, RelTraitSet traits, RelNode child) {
+  double percent;    // Sampling percent - a number between 0 and 1
```

Comment:
'percent' would imply a value between 0 and 100.    Otherwise, we should call it sampleFraction or sampleRatio.

Reply:
Kept it to percent and changed the range to 0-100